ParabolInc/action
# Retro UI Updates

This largely addresses #1975 plus a lot of refinements to clean up and represent original intent.

Also, it adds the “quick-and-easy” fix for #1976.

#### To Test
- Run a test with 2 local users
- Complete the primary actions in each phase
- Check out the summary in browser and email
- Report unusable or really terrible looking UI issues
